sdk/identity/identity/src/credentials/interactiveBrowserCredentialOptions.ts

```diff
@@ -63,7 +63,44 @@ export type InteractiveBrowserCredentialBrowserOptions = TokenCredentialOptions
      * Specifies whether a redirect or a popup window should be used to
      * initiate the user authentication flow. Possible values are "redirect"
      * or "popup" (default) for browser and "popup" (default) for node.
-     *
      */
     loginStyle?: BrowserLoginStyle;
+
```

Comment:
nitpick: to me "log in" refers to the human action of going through the process of providing credentials to a service, and I'm not sure it's the right verb here. What about "oidcState" and "oidcNonce" to make it clear that these are a way to pass options down to the Open ID Connect protocol layer.

Reply:
I do like the idea of not using login! Let me think about this

Reply:
I’ve moved these options to a property that is an object, called `loginOptions`. Also, for the `loginOptions` docs I’ve put a mention to the Open ID Connect protocol layer. So, the reference can be seen while the code remains readable!